File: app/src/main/java/fr/socket/florian/dhome/network/model/Tokens.java
package fr.socket.florian.dhome.network.model;

import com.google.gson.annotations.SerializedName;

public class Tokens extends ResponseBase {

    @SerializedName("sessionToken")
    private String sessionToken;

    @SerializedName("refreshToken")
    private String refreshToken;

    public String getSessionToken() {
        return sessionToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }
}

File: app/src/main/java/fr/socket/florian/dhome/network/model/Tokens.kt
package fr.socket.florian.dhome.network.model

import com.google.gson.annotations.SerializedName

class Tokens : ResponseBase() {

    @SerializedName("sessionToken")
    val sessionToken: String? = null

    @SerializedName("refreshToken")
    val refreshToken: String? = null
}
